object-manipulation-2/src/defaults.js

```diff
@@ -0,0 +1,26 @@
+/* exported defaults */
+/*
+set counter to 0
+use a for in loop for the source
+  set counter = 0 for each loop
+  use a for in loop for the target
+    check if items in the target is equal to the one item in the source
+      if yes, counter ++
+    if counter is 0, then the item is not in the target
+      add the item
+*/
+
+function defaults(target, source) {
+  var counter = 0;
```

Comment:
Instead of using a number for this, I would use a boolean, since you will only ever have values of `0` or `1`